lib/customer-deposit-wallet/customer-deposit-wallet.cabal

```
@@ -79,6 +80,7 @@ library
     Cardano.Wallet.Deposit.IO.Network.Mock
     Cardano.Wallet.Deposit.IO.Network.Type
     Cardano.Wallet.Deposit.IO.Resource
+    Cardano.Wallet.Deposit.IO.Resource.Dispatch
```

```suggestion
    Cardano.Wallet.Deposit.IO.Resource.Event
```

The style / paradigm is called [event-driven programming](https://en.wikipedia.org/wiki/Event-driven_programming). Maybe adjust the name accordingly? 🤔